src/py/pyodide/_package_loader.py

```diff
@@ -22,8 +23,14 @@
 from ._core import IN_BROWSER, JsProxy, to_js
 
 SITE_PACKAGES = Path(getsitepackages()[0])
-STD_LIB = Path(sysconfig.get_path("stdlib"))
-TARGETS = {"site": SITE_PACKAGES, "lib": STD_LIB, "dynlib": Path("/usr/lib")}
+if sys.base_prefix != sys.prefix:  # Are we in a virtualenv?
+    STD_LIB = Path(sysconfig.get_path("stdlib"))
+    TARGETS = {"site": SITE_PACKAGES, "lib": STD_LIB, "dynlib": Path("/usr/lib")}
+else:
+    # Better not put stuff into /usr/lib or /lib/python3.10! For now let's stick
+    # everyone into SITE_PACKAGES in this case
+    TARGETS = {"site": SITE_PACKAGES, "lib": SITE_PACKAGES, "dynlib": SITE_PACKAGES}
```

Comment:
Is this because we mount host `/usr` directory when using a command line runner?

Reply:
Yeah. We want to load them permanently but not into global user directories. If we didn't mount `/usr` then we'd also have the problem that we'd install them into memory and throw them away. Maybe there is a more appropriate path for libraries inside the virtual environment.

Reply:
Actually I think the emfs `/usr/lib` may shadow the native one.

Reply:
Okay, maybe we can think of installing all libraries into site-packages even if we are not in virtualenv, which is reverting the change that I made in #2721.

IIRC why I changed the installation directory was that shapely was trying to find geos from '/usr/lib'. But I think I can try to make a fix in #2954.